about.md

```diff
@@ -123,10 +111,13 @@ the development of OpenSSL's QUIC functionality. As a strong believer in the
 values of open source, I'm proud to be part of the OpenSSL project given its
 instrumental role in the security of internet infrastructure.
 
+## Kajal Sapkota
+To Be Defined =)
+
 ## Kurt Roeckx
 To Be Defined =)
 
-## Mark J Cox
+## Mark J Cox (emeritus)
```

Comment:
Should the text be updated as well?

Reply:
Or maybe we should have a separate section/page listing all notable past members of the team?